.github/scripts/generate_ci_workflows.py

```diff
@@ -115,25 +115,6 @@ def __post_init__(self) -> None:
         if self.is_libtorch:
             self.exclude_test = True
 
-        # The following code allows for scheduled jobs to be debuggable by
-        # adding the label 'ciflow/scheduled' and assigning + unassigning pytorchbot,
-        # without overwriting the workflow's own ciflow_config, if already enabled.
-        if self.is_scheduled:
-            if self.ciflow_config.enabled:
-                self.ciflow_config.labels.add('ciflow/scheduled')
-            else:
-                self.ciflow_config = CIFlowConfig(
-                    enabled=True,
-                    labels={'ciflow/scheduled'}
-                )
-
-        # CIFlow requires on_pull_request to be set in order to be enabled.
-        # If on_pull_request wasn't previously specified, we set trigger_action_only
-        # to True as we want to avoid unintentional pull_request event triggers
-        if self.ciflow_config.enabled:
-            self.ciflow_config.trigger_action_only = self.ciflow_config.trigger_action_only or not self.on_pull_request
-            self.on_pull_request = True
-
```

Comment:
wanted to simplify the logic in the __post_int__, and delegate the settings back to individual workflows, this gives us more flexibility in changing individual workflows, as well as reducing the complexity of understanding the mutation conditions.